src/components/ActivityFeed/LatestActivity.tsx

```
@@ -0,0 +1,316 @@
+						.toLowerCase();
+
+					return formattedTrackName === currentTab;
+			  });
+
+	return (
+		<div className=''>
+			{loading ? (
+				<div className='flex min-h-[200px]  items-center justify-center rounded-lg bg-white px-5 dark:bg-[#1c1d1f]'>
+					<Skeleton active />
+				</div>
+			) : currentuser && currentuser?.id && currentuser?.username ? (
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)